src/commands/notify.yml

```diff
@@ -71,6 +86,9 @@ steps:
         SLACK_PARAM_TAGPATTERN: "<<parameters.tag_pattern>>"
         SLACK_PARAM_CHANNEL: "<<parameters.channel>>"
         SLACK_PARAM_IGNORE_ERRORS: "<<parameters.ignore_errors>>"
+        SLACK_ICON: "<<parameters.icon>>"
```

Comment:
Orb job level env vars should be prefixed with the same pattern as the other environment variables, prefixed with `SLACK_PARAM_`

example: `SLACK_ICON` -> `SLACK_PARAM_ICON`